src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,23 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeModal();
+    }
+
+    public void closeModal() {
+        WebElement closeButton = browser.driver.findElement(By.className("close"));
+        waitForElementToBeClickable(closeButton);
+        click(closeButton);
+        try {
+            Thread.sleep(1000);
```

Comment:
@chowyb noted. For now would it be okay for me to replace the`Thread.sleep(1000)` with `waitForElementToDisappear(By.id("more-info-equal-share-modal")`? Just so that we don't further hinder whoever will be working on the PR for Issue #3317 but also don't make this PR wait for the eventual merger of that PR which hasn't even been created?